filament/backend/include/private/backend/Driver.h

```diff
@@ -55,6 +55,8 @@ class Driver {
 
     virtual Dispatcher& getDispatcher() noexcept = 0;
 
+    virtual void getCapabilities(RenderCapabilities& capabilities) const noexcept = 0;
```

Comment:
Should this return a reference to owned data instead of filling out the passed in structure?  I chose the latter to be minimally intrusive to filament.